### PR TITLE
[Claim - Approve Page] - UI/UX improvement: errors, messages, minor styles

### PR DESCRIPTION
**Pull request by W3stside**
# Summary

Addresses the issue in our doc:
image

## IMPORTANT!
- this inclides "REVOKE" but that functionality is only rendered in testing environments, meaning: PRaul, local, cowswap.dev.
- const for this can be checked in the file with the CLAIM const: pages/Claim/const.ts

## Test
1. go to investment approving page
2. make sure u need to approve a token
3. see error to approve
4. hit approve
5. see error saying wait for approve

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @W3stside , nice!
But could we show somehow a 'approving' message instead of the loader on the approve button OR instead of the 'Token not approved' message? 
image

WDYT?

**Comment by W3stside** [comment]
> Hey @W3stside , nice! But could we show somehow a 'approving' message instead of the loader on the approve button OR instead of the 'Token not approved' message? image
> 
> WDYT?

I could change "GNO not approved"' to "Approving GNO..." and keep loader

**Comment by W3stside** [comment]
@elena-zh this has been updated to reflect what you requested.

**Comment by elena-zh** [comment]
@W3stside , looks great to me!

Question: will 'revoke' feature be disabled after merge? 
It works great, but 'revoking' state is almost invisible in the light mode
almost invisible

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx [hunk 1]
```diff
@@ -123,8 +121,16 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
     setTypedValue(value.toExact() || '')
   }, [balance, maxCost, noBalance])
 
-  // Save "local" approving state (pre-BC) for rendering spinners etc
+  // Save "local" approving state for rendering spinners etc
   const [approving, setApproving] = useState(false)
+  const isApproved = approveState === ApprovalState.APPROVED
+  const notApproved = approveState !== ApprovalState.APPROVED
```
Comment (alfetopito) [inline src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:127]: Really? What's wrong with using `!isApproved`?

**Review by alfetopito — APPROVED** [review]
Looks good overall, just one variable seems unnecessary

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx (hunk 1 above)
Comment (W3stside) [reply src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:127]: Haha shh

**Review by elena-zh — APPROVED** [review]
Approved, but still unclear about this comment: https://github.com/gnosis/cowswap/pull/2304#issuecomment-1022374566

File: src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx [hunk 2]
```diff
@@ -304,30 +315,25 @@ export default function InvestOption({ claim, optionIndex, openModal, closeModal
...
-              // CURRENTLY TEST ONLY
-              approveState === ApprovalState.APPROVED && (
-                <UnderlineButton disabled={approving || isPendingApproval} onClick={handleRevokeApproval}>
-                  Revoke approval {approving || (isPendingApproval && <Loader size="12px" stroke="white" />)}
+            {
+              // CURRENTLY TEST ONLY (not on prod, barn or ens)
+              IS_TESTING_ENV && isApproved && (
+                <UnderlineButton disabled={approving || isPendingRevoke} onClick={handleRevokeApproval}>
+                  {(approving || isPendingRevoke) && <Loader size="12px" stroke="white" />}
+                  {approving || isPendingRevoke ? 'Revoking approval . . .' : 'Revoke approval'}{' '}
```
Comment (anxolin) [inline src/custom/pages/Claim/InvestmentFlow/InvestOption.tsx:333]: nit: Why did you add those dots like that? we usually use the tree consecutive dots `...`

Looks a bit strange to me:
image

**Review by anxolin — APPROVED** [review]

**Comment by W3stside** [comment]
> @W3stside , looks great to me!
> 
> Question: will 'revoke' feature be disabled after merge? It works great, but 'revoking' state is almost invisible in the light mode almost invisible

it'll always be present (probably) only in dev environments, so nothing in production (ens or barn) - will address colours
